.clang-tidy

```diff
@@ -1,14 +1,22 @@
 Checks: '-*,readability-identifier-naming'
 CheckOptions:
 - key:             readability-identifier-naming.PrivateMemberPrefix
-  value:           m_
+  value:           'm_'
+- key:             readability-identifier-naming.ClassConstantCase
+  value:           aNy_CasE
+- key:             readability-identifier-naming.ClassConstantPrefix
```

Comment:
`readability-identifier-naming.ClassConstantPrefix` is a specialisation of `readability-identifier-naming.ConstantMemberPrefix`. So you only need to change one of these keys (`readability-identifier-naming.ClassConstantPrefix` to only set the prefix for `static` members).